### ISSUE TITLE
[Mac][Accessibility] NSCombox doesn't allow focus with keyboard

### ISSUE THREAD

**Issue by netonjm** [xamarin/Xamarin.PropertyEditing#497]
When we are navigating with tab in the controls, some items (NSComboBox) are not in the Focus chain.

Any control should be focusable with the keyboard.

This happens with the properties and with the ArrangleBy combo.

![focusissue](https://user-images.githubusercontent.com/1587480/51377324-ea894b80-1b0a-11e9-8092-1332221c4dc1.gif)

**Comment by CartBlanche** [comment]
Unless the ComboBox is set to `Editable` it doesn’t accept focus, AFAICT. This can be seen in XCode. So Mac users should be familiar with that behaviour. Should it work better, absolutely. We would need to investigate if there is some simple tweaks to `NSComboBox` we can do to get that behaviour, before we go down the route or writing a totally custom ComboBox

**Comment by ermau** [comment]
The brush editor uses a combo box doesn't it? It's focusable.
